Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -242,34 +242,24 @@ static void EncodeToRamUsingShader(GLuint srcTexture, u8* destAddr, u32 dst_line
 
   int dstSize = dst_line_size * dstHeight;
 
-  if ((writeStride != dst_line_size) && (dstHeight > 1))
+  // When the dst_line_size and writeStride are the same, we could use glReadPixels directly to RAM.
+  // But instead we always copy the data via a PBO, because macOS inexplicably prefers this for some
+  // reason.
+  glBindBuffer(GL_PIXEL_PACK_BUFFER, s_PBO);
+  glBufferData(GL_PIXEL_PACK_BUFFER, dstSize, nullptr, GL_STREAM_READ);
+  glReadPixels(0, 0, (GLsizei)(dst_line_size / 4), (GLsizei)dstHeight, GL_BGRA, GL_UNSIGNED_BYTE,
```

Comment:
All our textures are in GL_RGBA order, so to skip the byte swap in the readback I would do this as RGBA, and instead swap the byte order in the shader. But this is out of the scope of this PR.